xclim/core/formatting.py

```diff
@@ -565,3 +566,47 @@ def generate_indicator_docstring(ind):
 
     doc = f"{header}\n{special}\n{parameters}\n{returns}\n{extras}"
     return doc
+
+
+def publish_release_notes(style: str = "md") -> str:
```

Comment:
I feel this should be in `xclim.testing`, no? With other functions with similar scopes?